### DISCUSSION TITLE
Expose ElFocusTrap component

### DISCUSSION THREAD

**Discussion by hellodresji** [element-plus/element-plus#10282] [Feature Request]
In some cases the ElFocusTrap also needs to be used for our custom components, but it cannot be imported, because it is not exposed I think. 
<img width="611" alt="image" src="https://user-images.githubusercontent.com/13019006/198222599-f80dd607-63c5-4f88-8427-6e9d0ef855ff.png">

**Comment by ryuhangyeong** [comment]
@jw-foss How about this idea?
